accounts/permissions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import permissions
+
+
+class IsOwner(permissions.BasePermission):
+    """ 
+    custom permission to only allow owners of object to edit, delete or update
+    """
```

Comment:
clean approach

Reply:
Accounts details belong to a particular user, so whenever a post is created who do you want to associate that post with? Is it the account or the user? Tell me how will you relate the account and the token? Think it through even without referring to the ERD we were given..

Reply:
Won't that create an issue, say a user has 3 accounts how will you specify if the deletion is of a specific post in one particular account. The user should have the ability to create or delete an account, but only the account has access to the blogs, simple is better.

Reply:
> Won't that create an issue, say a user has 3 accounts how will you specify if the deletion is of a specific post in one particular account. The user should have the ability to create or delete an account, but only the account has access to the blogs, simple is better.

I think the deletion can be specified by the corresponding foreign keys/primary keys.